inc/lift/RequestHandle.h

```diff
@@ -79,6 +79,17 @@ class RequestHandle
         std::chrono::milliseconds timeout
     ) -> bool;
 
+    /**
+     * Sets the maximum number of bytes of data to write.
+     * (The number of bytes downloaded may be greater than the set amount,
+     * but the number of bytes written for the response's data will not
+     * exceed this amount.)
+     * @param max_download_bytes     The maximum number of bytes to be written for this request.
+     */
+    auto SetMaxDownloadBytes(
+            ssize_t max_download_bytes
```

Comment:
nitpicking, but this looks like its tabbed in twice

Reply:
Fixed!